docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -7,16 +7,16 @@ pre = "<b> </b>"
 
 Starting with release 2.2, the operator supports FMW Infrastructure domains.
 This means domains that are created with the FMW Infrastructure installer rather than the WebLogic
-installer.  These domains contain the Java Required Files (JRF) feature and are
+Server installer.  These domains contain the Java Required Files (JRF) feature and are
 the pre-requisite for "upper stack" products like Oracle SOA Suite, for example.
 These domains also require a database and the use of the Repository
 Creation Utility (RCU).
 
 This section provides details about the special considerations for running
 FMW Infrastructure domains with the operator.  Other than those considerations
-listed here, FMW Infrastructure domains work in the same way as WebLogic domains.
+listed here, FMW Infrastructure domains work in the same way as WebLogic Server domains.
```

Comment:
I saw but didn't make this correction originally (from WebLogic domains -> WebLogic Server domains), but I'm glad that you did. From the standpoint of the official documentation, it's more accurate and technically correct.